psi4/src/read_options.cc

```diff
@@ -109,7 +109,7 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
     of orbitals to freeze, program picks the lowest-energy orbitals)
     or |globals__frozen_docc| (gives the number of orbitals to freeze per
     irreducible representation) -*/
```

Comment:
This string is parsed and included in the manual to document the keyword, so this is the most important place to describe the behavior of the different values, in case you didn't know that already